$NetBSD: patch-ac,v 1.4 2000/09/10 20:59:07 wiz Exp $

If libnet is installed, this check results in a link error. -- wiz

--- configure.orig	Sat Apr 10 18:02:18 1999
+++ configure
@@ -2499,46 +2499,6 @@
   echo "$ac_t""no" 1>&6
 fi
  # SVR4 sockets
-echo $ac_n "checking for socket in -lnet""... $ac_c" 1>&6
-echo "configure:2504: checking for socket in -lnet" >&5
-ac_lib_var=`echo net'_'socket | sed 'y%./+-%__p_%'`
-if eval "test \"`echo '$''{'ac_cv_lib_$ac_lib_var'+set}'`\" = set"; then
-  echo $ac_n "(cached) $ac_c" 1>&6
-else
-  ac_save_LIBS="$LIBS"
-LIBS="-lnet $LIBS $LIBS"
-cat > conftest.$ac_ext <<EOF
-#line 2512 "configure"
-#include "confdefs.h"
-/* Override any gcc2 internal prototype to avoid an error.  */
-/* We use char because int might match the return type of a gcc2
-    builtin and then its argument prototype would still apply.  */
-char socket();
-
-int main() {
-socket()
-; return 0; }
-EOF
-if { (eval echo configure:2523: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
-  rm -rf conftest*
-  eval "ac_cv_lib_$ac_lib_var=yes"
-else
-  echo "configure: failed program was:" >&5
-  cat conftest.$ac_ext >&5
-  rm -rf conftest*
-  eval "ac_cv_lib_$ac_lib_var=no"
-fi
-rm -f conftest*
-LIBS="$ac_save_LIBS"
-
-fi
-if eval "test \"`echo '$ac_cv_lib_'$ac_lib_var`\" = yes"; then
-  echo "$ac_t""yes" 1>&6
-  LIBS="-lnet $LIBS"
-else
-  echo "$ac_t""no" 1>&6
-fi
- # BeOS
 ;;
 esac
 
